filament/backend/src/metal/MetalHandles.mm

```diff
@@ -816,6 +823,13 @@ BufferDescriptor b(nullptr, 0u, [](void* buffer, size_t size, void* user) {
                 "MetalRenderTarget was initialized with a MSAA COLOR%d texture, but sample count is %d.",
                 i, samples);
 
+        auto t = color[i].metalTexture;
+        const auto twidth = std::max(1u, t->width >> color[i].level);
+        const auto theight = std::max(1u, t->height >> color[i].level);
+        tmin = { std::min(tmin.x, twidth), std::min(tmin.y, theight) };
+        tmax = { std::max(tmax.x, twidth), std::max(tmax.y, theight) };
+        attachmentCount++;
+
```

Comment:
this min/max method is good, but I think we can do better, you can check that the bitwise OR and the bitwise AND of all the values are the same. You can use the fact that:
if a==b, then (a&b) == (a|b) and by extension
if a==b and b==c, then (a&b&c) == (a|b|c)